tools/tidb-binlog-kafka.md

```diff
@@ -107,206 +121,212 @@ Kafka 集群用来存储由 Pump 写入的 binlog 数据，并提供给 Drainer
   ``` 
 
 #### 使用 Binary 部署 PUMP
-
-1. PUMP 命令行参数说明
-
+    使用样例：
+    假设我们有三个 PD，三个 zookeeper，一个 TiDB, 各个节点信息如下  
     ```
-    Usage of pump:
-    -L string
-        日志输出信息等级设置: debug, info, warn, error, fatal (默认 "info")
-    -V
-        打印版本信息
-    -addr string
-        pump 提供服务的 rpc 地址(默认 "127.0.0.1:8250")
-    -advertise-addr string
-        pump 对外提供服务的 rpc 地址(默认 "127.0.0.1:8250")
-    -config string
-        配置文件路径,如果你指定了配置文件，pump 会首先读取配置文件的配置
-        如果对应的配置在命令行参数里面也存在，pump 就会使用命令行参数的配置来覆盖配置文件里面的
-    -data-dir string
-        pump 数据存储位置路径
-    -kafka-addrs string
-        连接的 kafka 的地址 (默认 "127.0.0.1:9092")
-    -zookeeper-addrs string
-        zookeeper 地址，如果设置了该选项则从 zookeeper 中获取 kafka 地址，如果不设置则使用 kafka-addrs 的值
-    -gc int
-        日志最大保留天数 (默认 7)， 设置为 0 可永久保存
-    -heartbeat-interval uint
-        pump 向 pd 发送心跳间隔 (单位 秒)
-    -log-file string
-        log 文件路径
-    -log-rotate string
-        log 文件切换频率, hour/day
-    -metrics-addr string
-       prometheus pushgataway 地址，不设置则禁止上报监控信息
-    -metrics-interval int
-       监控信息上报频率 (默认 15，单位 秒)
-    -pd-urls string
-        pd 集群节点的地址 (默认 "http://127.0.0.1:2379")
-    -socket string
-        unix socket 模式服务监听地址 (默认 unix:///tmp/pump.sock)
+    TiDB="192.168.0.10"
+    PD1="192.168.0.16"
+    PD2="192.168.0.15"
+    PD3="192.168.0.14"
+    ZK1="192.168.0.13"
+    ZK2="192.168.0.12"
+    ZK3="192.168.0.11"
     ```
+    在 ip="192.168.0.10" 的机器上面部署 drainer pump
+    对应的 pd 集群的 ip="192.168.0.16,192.168.0.15,192.168.0.14" 
+    对应的 kafka 集群的 zookeeper 的 ip="192.168.0.13,192.168.0.12,192.168.0.11" 以此为例，说明 pump drainer 的使用
```

Comment:
Capitalize the first letter of the component names:

Drainer, Pump, PD, Kafka, Zookeeper

Pls update similar items involved in this PR.